odxtools/compumethods/readcompumethod.py

```diff
@@ -47,7 +47,7 @@ def _parse_compu_scale_to_linear_compu_method(scale_element,
 
     offset = computation_python_type(next(nums).text)
     factor_el = next(nums, None)
-    factor = computation_python_type(factor_el.text if factor_el else '0')
```

Comment:
Hi @andlaus - The previous MR https://github.com/mercedes-benz/odxtools/pull/53 causes the numerator to default to 0, when there are more than one `V` in numerator. 

`if factor_el` does not return `True`, making it an explicit check.

Reply:
Fixed already in #56

Reply:
Thank you @kayoub5